tr/news/_posts/2019-04-01-ruby-2-4-6-released.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: news_post
+title: "Ruby 2.4.6 Yayınlandı"
+author: "usa"
+translator: "İsmail Arılık"
+date: 2019-04-01 06:00:00 +0000
+lang: tr
+---
+
+Ruby 2.4.6 yayınlandı.
+
+Bu yayın önceki yayından sonra 20 hata düzeltmesi içerir, ve ayrıca birkaç güvenlik düzeltmesi içerir.
+Ayrıntılar için lütfen aşağıdaki konuları gözden geçirin.
+
+* [RubyGems'te birçok güvenlik açığı](/tr/news/2019/03/05/multiple-vulnerabilities-in-rubygems/)
+
+Ayrıntılar için [işleme loguna](https://github.com/ruby/ruby/compare/v2_4_5...v2_4_6) bakın.
+
+Bu yayından sonra, Ruby 2.4'ün normal sürdürme evresini sona erdireceğiz, ve güvenlik sürdürme evresine başlayacağız.
+Bu, 2.4.6 yayınından sonra güvenlik düzeltmeleri dışında asla herhangi bir hata düzeltmesini 2.4'e geri taşımayacağımız anlamına geliyor.
+Güvenlik sürdürme evresi dönemi 1 seneye zamanlanır.
```

Comment:
bunu anlamadım

Reply:
Doğrudan çevirince böyle bir çeviri meydana geliyor. "Maintenance" sözcüğünü "sürdürme" olarak çevirdim. Orijinal metinde "normal maintenance" ve "security maintenance" ifadeleri var.

> Güvenlik sürdürme evresi dönemi 1 seneye zamanlanır.

Bu da doğrudan çeviri. "Scheduled" sözcüğünü "zamanlamak" olarak çevirdim.

Reply:
bence şu şekilde daha anlaşılır. `Güvenlik bakım aşamasının süresi 1 yıl için planlanmıştır.` çünkü burada asıl demek istedikleri o.

Reply:
c632d30